src/sequential/large_encryptor.rs

```diff
@@ -39,6 +39,7 @@ where
     // Constructor for use with pre-existing `DataMap::Chunks` where there are more than three
     // chunks.  Retrieves the first two and and last two chunks from storage and decrypts them to
     // its internal buffers.
+    #[allow(clippy::new_ret_no_self)]
```

Comment:
Maybe we can allow this crate-wide instead of five separate times throughout the crate. I don't personally like this lint anyway as it fails on a lot of real-world examples -- even the std lib has `new` functions that don't return `Self`.

Reply:
I was thinking that too. My reasoning was that we could want to encourage future `new` functions to return `Self`, but I'm with you that `new` is a good name for a function that returns a `Result<Self, _>`...
I'll deny it crate-wide.

Reply:
As discussed, we'll tackle this with the next CI PR (musl)